roadmap/implementors-guide/src/SUMMARY.md

```diff
@@ -36,7 +36,14 @@
     - [Network Bridge](node/utility/network-bridge.md)
     - [Misbehavior Arbitration](node/utility/misbehavior-arbitration.md)
     - [Peer Set Manager](node/utility/peer-set-manager.md)
-- [Data Structures and Types](type-definitions.md)
+	- [Runtime API Requests](node/utility/runtime-api.md)
```

Comment:
```suggestion
    - [Runtime API Requests](node/utility/runtime-api.md)
```

I think your editor is configured to add tabs, not spaces, even in markdown files. That's messing with the consistency within this file.